dataingest/spatial/geo2sql.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"log"
+	"os"
+
+	"github.com/ONSdigital/dp-find-insights-poc-api/dataingest/spatial/table"
+	"github.com/twpayne/go-geom/encoding/geojson"
+)
+
+// couldn't find this as a go-geom type?
+type FeatureCollection struct {
+	Type     string            `json:"type"`
+	Name     string            `json:"name"`
+	CRS      geojson.CRS       `json:"crs"`
+	Features []geojson.Feature `json:"features"`
+}
+
+func main() {
+	filename := flag.String("f", "", "geojson file to import")
+	tablename := flag.String("t", "", "name of table to create")
+	mode := flag.String("m", "copy", "SQL copy or insert")
+	keepcase := flag.Bool("k", false, "preserve case of property (column) names in features")
```

Comment:
worth making it clear here that you're getting lowercase if you leave this false?

Reply:
Yep, added a bit to the help text.